airflow/www/views.py

```diff
@@ -1320,76 +1320,7 @@ def legacy_dag_details(self):
     @provide_session
     def dag_details(self, dag_id, session: Session = NEW_SESSION):
         """Get Dag details."""
-        from airflow.models.dag import DagOwnerAttributes
```

Comment:
Please remove `@auth.has_access_dag("GET", DagAccessEntity.RUN)` on line 1319. Authz should be done on the final page in case of redirection

Reply:
Done!